docs/source/tutorials/configure_data.rst

```diff
@@ -0,0 +1,372 @@
+.. _how_to_configure_data:
+
+How To Configure Data
+=====================
+
+Olive Data config organizes the data loading, preprocessing, batching and post processing into a single json config and define several **popular templates** to serve olive optimization.
+
+The configuration of olive data config is positioned under olive run config with the field of `data_configs`, and the data config is defined as a list of data config items. Here is an example of data config: `open_llama_sparsegpt_gpu <https://github.com/microsoft/Olive/blob/main/examples/open_llama/open_llama_sparsegpt_gpu.json#L11-L49>`_ .
+
+.. code-block::
+
+    "data_configs": {
+        "dataset_1": {...},
+        "dataset_2": {...},
+    }
+
+Then if there is any requirement to leverage the data config in olive passes/evaluator, we can simply refer to the data config **key name**. For above `open_llama_sparsegpt_gpu` case, the passes/evaluator data config is:
+`open_llama_sparsegpt_gpu data_config reference <https://github.com/microsoft/Olive/blob/main/examples/open_llama/open_llama_sparsegpt_gpu.json#L59>`_ .
+
+.. code-block::
+
+    "evaluators": {
+        "common_evaluator": {
+            ...,
```

Comment:
would this introduce doc build warning?

Reply:
I do not add the `json` flag, so it is ok.